PendingReleaseNotes

```diff
@@ -48,6 +48,7 @@
   CephFS does not support disk space reservation. The only flags supported are
   `FALLOC_FL_KEEP_SIZE` and `FALLOC_FL_PUNCH_HOLE`.
 
+
```

Comment:
Don't get. Shouldn't we have a release note for squid minor?

Reply:
Right. Fixed.